ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dot1x.schema.yml

```diff
@@ -13,3 +13,17 @@ keys:
         type: bool
       dynamic_authorization:
         type: bool
+      mac_based_authentication:
+        type: dict
+        keys:
+          delay:
+            type: int
+          hold_period:
+            type: int
+      radius_av_pair:
+        type: dict
+        keys:
+          service_type:
+            type: bool
+          framed_mtu:
+            type: int
```

Comment:
please add
```
convert_types:
  - str
```
on all the `int` fields. This will allow a user to user inline jinja templating, which often returns a string.